functional.py

```diff
@@ -0,0 +1,62 @@
+import sys
+
+def sequential_map(*args):
+    object = args[-1]
+    if type(object) == int:
+        for func in args[:-1]:
+            object = func(object)
+    else:
+        for func in args[:-1]:
+            object = list(map(func, object))
+    return object
```

Comment:
Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
```suggestion
    else:
        for func in args[:-1]:
            object = map(func, object)
    return list(object)
```